src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -69,7 +69,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AssemblySearchPaths>{CandidateAssemblyFiles};{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
   </PropertyGroup>
+  
+  <!-- Default glob settings -->
+  <PropertyGroup>
+    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;packages\**</GlobalExclude>
+  </PropertyGroup>
 
+  <ItemGroup Condition=" '$(OverridDefaultGlobs)' != 'True' ">
```

Comment:
It didn't cause a test failure because I had copied and pasted the property name into the tests :-)